pkgs/by-name/mm/mmdbinspect/package.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "mmdbinspect";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "maxmind";
+    repo = "mmdbinspect";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-PYn+NgJDZBP+9nIU0kxg9KYT0EV35omagspcsCpa9DM=";
+    fetchSubmodules = true;
+  };
+
+  vendorHash = "sha256-HNgofsfMsqXttnrNDIPgLHag+2hqQTREomcesWldpMo=";
+
+  ldflags = [
+    "-s"
+    "-w"
+  ];
+
```

Comment:
Would be better to add https://nixos.org/manual/nixpkgs/unstable/#versioncheckhook

Reply:
sadly, no version command can be used.